poetry.lock

```diff
@@ -1,11 +1,12 @@
-# This file is automatically @generated by Poetry 1.8.5 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.0.1 and should not be changed by hand.
```

Comment:
¯\\\_(ツ)\_/¯ I just happen to have Poetry 2.0.1 installed instead of 1.8.5. Meanwhile the [current version is 2.1.1](https://pypi.org/project/poetry/)